ui/start_ui.py

```diff
@@ -794,23 +852,24 @@ def set_rag_default_path(self, selector, rag_path):
 
     def _init_ui(self):
         mark_alive = None
+        private_key = paramiko.Ed25519Key.from_private_key_file("/root/.ssh/id_ed25519")
```

Comment:
Why use a custom path to set private key? `self.ssh_connect[index].load_system_host_keys()` can load the content of  '~/.ssh/known_hosts'  by default.

Reply:
OK, I see. I'm not sure about the format of your known_hosts, could you verify this method in your env?
For example, the node ip is `10.1.0.133`, then run `ssh-keyscan 10.1.0.133 >> ~/.ssh/known_hosts`. And then modify `10.1.0.133`  to `[10.1.0.133]:22` like this:
<img width="299" alt="image" src="https://github.com/intel/llm-on-ray/assets/97155466/31c90f39-8f6a-49bc-9e45-d0808b232c14">
If this doesn't work, it really needs a fallback method.